### DISCUSSION TITLE
[Projects] Feedback on GitHub projects Beta Side-Panel

### DISCUSSION THREAD

**Discussion by omerbensaadon** [community/community#15350] [Projects and Issues] [answered]
This is a feedback Discussion for users of the GitHub Projects Side-Panel

Leave your feature requests below!

**Comment by abhimskywalker** [comment]
@omerbensaadon how to enable/access this in my github project? 🤔
I was about to comment that iPad like side-Panel for issue navigation feels very nice and would make Github Projects experience almost like any of the modern project management tools like asana, etc. 🚀
PS: If this is not what I am talking about, apologies in advance 😅

**Reply by omerbensaadon** [reply]
Going to enable this as a Feature Preview early this week, sorry for the delay!

**Reply by abhimskywalker** [reply]
Works nicely. 
I guess non-issue tasks will also become first class citizens with this :)
Timeline is the only thing missing, but description and comments from kanban style view is great.

Feedback/UX-break:
👉 From List view when i click on issues they take me to new tab with issue page, but non-issue tasks open in the side-panel. Perhaps a uniform experience for side-panel opening on all clicks would be nice to have here

**Reply by vpriya** [reply]
looks nice

**Comment by mvstepanek** [comment]
I like that feature. Unfortunately, for now, it's not working with the Grammarly extension. If I want to correct something, I get a popup with `You have unsaved changes. Press cancel to continue editing.`.

**Reply by Julian-Dumitrascu** [reply]
It works with a similar extension.

**Comment by thibaudcolas** [comment]
It works exactly as I would have expected it to. With this in place, it feels much more practical to stay within projects’ Kanban-style Board view, rather than having to switch back to Table view whenever we have to make edits.

**Comment by vojtech-dobes** [comment]
In my opinion this is spot on implementation 🎆, works exactly as I would like it to. 💯 

Just slightly related thing (that I've encountered already at regular GitHub issue detail page) is that showing all fields by default for a project would be much more ergonomic than having to display them with extra click. Especially if issue is part of a single project only (our case in 99% of issues).

**Reply by omerbensaadon** [reply]
thanks for your feedback! Development on the side-panel will resume in a few weeks once we've had the opportunity to collect feedback 👀 will keep this in mind

**Reply by dmackerman** [reply]
Yes, this is a major annoyance for our team because we have 7+ fields that we're tracking!

**Reply by LukeWolfGames** [reply]
I also did post about this, but saw this late. Thanks for also sharing this. Really appreciate it and I also think that the side panel should be displaying more consistently - a lot like the status field instead of displaying them differently and taking up more room: 
![image](https://user-images.githubusercontent.com/28833199/172990755-444b5072-6513-45ee-b1d0-c0c78a3f240a.png)

**Comment by ReedOnly** [comment]
I had this feature yesterday, however today GitHub Projects opens a new tab instead of the side panel when clicking an issue. How can it be re-enabled? It worked wonderful for the few hours I had it! 🥳

**Reply by omerbensaadon** [reply]
Strange... Do you have it enabled in Feature Preview? https://docs.github.com/en/get-started/using-github/exploring-early-access-releases-with-feature-preview

**Reply by ReedOnly** [reply]
Now it works!! thanks so much 😍

**Comment by ch1bo** [comment]
Works perfectly for us, thanks!

One minor follow-up maybe: the "convert checkbox to subtask" feature would be great to have from the side-bar! Ideally with creating sub-task issue and directly adding it to the same project.

**Reply by omerbensaadon** [reply]
thanks for your feedback! Development on the side-panel will resume in a few weeks once we've had the opportunity to collect feedback 👀 will keep this in mind

**Comment by Kucharssim** [comment]
I love the side panel feature!

It would be great if there was an option to transfer/add an issue to a different project from the side panel. I did not find how to do that while staying in the current project page. So I assume that currently it's only possible by opening the issue in a new tab and reassign the issue to a different project from there.

**Reply by omerbensaadon** [reply]
thanks for your feedback! Development on the side-panel will resume in a few weeks once we've had the opportunity to collect feedback 👀 will keep this in mind

**Comment by samwyness-district** [comment]
Love the new side panel feature, well done GitHub team 👏 🚀

**Reply by omerbensaadon** [reply]
This totally exists in the table view! 

Just enable it and click any row

**Reply by samwyness-district** [reply]
It sure does! Thanks @omerbensaadon.
It wasn't working for me on the day the feature was released, even with the feature preview enabled.

**Reply by Fanweilin1** [reply]
nice！

**Comment by samwyness-district** [comment]
A really handy feature of issues is the 'Development' section on the issue page. Creating a branch via the issue directly is something we do in our project as a preferred practice (e.g. `feature/1234-create-an-awesome-side-panel`) as we often reference the issue number in discussions.

Enabling this feature in the Issue Side-Panel would allow us to use the Side-Panel for almost all of our efforts when managing an issue from the Project board 👌

<img width="321" alt="image" src="https://user-images.githubusercontent.com/88999537/167063893-a687d978-581d-4e7a-9702-507d85ed60b3.png">

**Reply by samwyness-district** [reply]
Discussion: https://github.com/github/feedback/discussions/16131

**Reply by askpt** [reply]
I agree with you! We should be able to quickly create branches from the new view. It's a great productivity feature!

**Reply by omerbensaadon** [reply]
thanks for your feedback! Development on the side-panel will resume in a few weeks once we've had the opportunity to collect feedback 👀 will keep this in mind

**Reply by zakaryarashak** [reply]
16131

**Reply by cory-baker** [reply]
@omerbensaadon Are you able to confirm if this is apart of current development?

**Comment by kithrup** [comment]
I also love how, even though it's disabled, it's still there.

**Reply by omerbensaadon** [reply]
hopefully this is resolved?

**Comment by jessmartin** [comment]
I'd really like to be able to deeplink to specific "Draft" Issues. I would think a simple implementation would be adjusting the URL when the sidebar slides out. i.e., `https://github.com/orgs/YourOrg/projects/17/views/1/drafts/1`

But I suspect the deeper issue is that these Draft-not-quite-Issues don't actually have publicly-visible IDs the way that Issues do! 😢 

In any case, I need to be able to pass a URL to someone and say "please look at this Draft issue" and that's currently not possible, unless I'm missing something.

**Reply by omerbensaadon** [reply]
Deep linking is definitely something we want to look at in the long-run, got a few other loose ends to tie up but I would expect this out of the product _eventually_

**Reply by drewvanstone** [reply]
We're running into this issue as well. We don't want to pollute the repo's Issues with subtasks, so we use draft issues. It would be very helpful to be able to link to these draft issues.

**Reply by vbalazs** [reply]
I think this is relevant also for issues, not only drafts.
It already happened several times in our team that they open an issue on the Side-Panel view, then copy-paste the URL and send it as a reference to the issue which is currently just a link to the view and not the card/issue.

**Reply by mattcosta7** [reply]
https://github.blog/changelog/2023-01-05-github-issues-january-5th-update/

We recently shipped urls that link directly to the projects sidepanel open to an issue or draft issue

**Comment by Vistaus** [comment]
Hmm, I enabled this in Feature Preview, but I don't see it. Is this an A/B test or something?

**Reply by MithrilMan** [reply]
because it has been poorly explained how to enable and use it...

First you need to have a Project(beta) in any of your repository
![image](https://user-images.githubusercontent.com/5107375/167454684-e99a3265-40da-4dde-a60f-8b3afae73dd1.png)

Select your (beta) project 
![image](https://user-images.githubusercontent.com/5107375/167455323-107b1b53-a78a-4e28-bcfe-f5c8734cc9f1.png)

Then select any of your issue 
![image](https://user-images.githubusercontent.com/5107375/167456049-d8af891a-4f4f-4888-9eaa-285a65dc3dad.png)

A panel will show up on the right, instead of a new browser tab with your clicked issue
![image](https://user-images.githubusercontent.com/5107375/167455730-da166190-9c12-4447-ba38-b8454f519f3b.png)

**Reply by MithrilMan** [reply]
In case you were already on the project page, you need to refresh the browser

**Reply by PrayagS** [reply]
Thanks for sharing this @MithrilMan. I think the GH team should update the note at the feature preview page with these instructions.

**Reply by jeffstedt** [reply]
+1

The feature preview description was very poor and I basically had to find my way here and read this comment to understand the new alpha feature.

**Reply by omerbensaadon** [reply]
Thanks for the feedback folks, we've enabled this by default globally but will certainly take this as feedback for myself to do a bette job writing these descriptions 😅

**Comment by yegeniy** [comment]
Fantastic idea. Please add the same kind of preview for PRs (even if only the "Conversation" tab) and it's a win!

**Reply by garrettm** [reply]
Agreed, having this on PRs would be really useful. Right now I am still a bit scared to click without thinking because sometimes it will be a PR and open a new tab.

**Reply by nul800sebastiaan** [reply]
Aha! That's why it wasn't working for me, it's only for issues, PRs still open in a new tab. I actually use project boards mostly for PRs so this feature is unfortunately not very useful for me yet. +1 for PR sidepanels!

**Reply by omerbensaadon** [reply]
thanks for your feedback! Development on the side-panel will resume in a few weeks once we've had the opportunity to collect feedback 👀 will keep this in mind 

PRs are probably a bit behind other changes and optimizations we're hoping to make, but rest assured we've got them in mind

**Reply by stevenjmesser** [reply]
PRs in the side panel would be incredibly useful for our stand-ups and sprint reviews. We don't have issues for everything, particularly external contributions, so we have to open new tabs for looking at comments on PRs.

**Comment by MithrilMan** [comment]
Definitively needed, thanks.

**Comment by fabiangeisler** [comment]
Love the feature! 🙂 
Is there a hotkey to open it from board or kanban views? I could not find one. Would be nice to move around the issues with the arrow keys and toggle the panel on or off as needed.

**Reply by iansan5653** [reply]
Hi @fabiangeisler - to open the side panel from board view, you can press <kbd>Enter</kbd> while focused on an item title.

**Reply by fabiangeisler** [reply]
Thanks @iansan5653 !
That does work, but is a bit cumbersome to get to. If you are only using the keyboard you have to hit `tab` quite a few times to get there. Would be great to have a short cut that opens the issue while any part of it is selected. Also after closing the panel, the selection is lost, which makes it hard to go through the board view fluently. 
Anyway, thanks for your hard work on this! 🙂

**Reply by fabiangeisler** [reply]
Just saw the release notes and that you added support to press <kbd>spacebar</kbd> for opening the issue site panel! This is great! Thank you! :smile: One minor additional request: Can we enable this behaviour for the kanban views as well?

**Comment by YurinDoctrine** [comment]
Hey can we do the same for .mds to the side instead bottom?

**Reply by omerbensaadon** [reply]
not totally clear what you mean here, we will eventually work on adding support for other kinds of objects (not just issues) in that side-panel but it's a bit down the road

**Comment by sammachin** [comment]
This is great for reviewing a number of items on a board, but;
I'd like to be able to change/edit the project (and status) that an issue is attached to, one of our workflows is to move items from our "Product" board to "Development" at the start of a sprint, and now its even more clicks to get to that.

**Reply by omerbensaadon** [reply]
this seems out of scope for side-panel feedback and sounds more like automation feedback?

**Comment by vojtech-dobes** [comment]
One thing I've noticed: "Tracked in" badge isn't displayed, which leads to necessity to open the full page detail anyway to find out.

**Reply by omerbensaadon** [reply]
thanks for your feedback! Development on the side-panel will resume in a few weeks once we've had the opportunity to collect feedback 👀 will keep this in mind

Tracked In is certainly something we hope to build in soon

**Comment by 00000-hello** [comment]
I think the side-panel looks a bit cluttered.
In my opinion, a compact view would be good.

**Reply by omerbensaadon** [reply]
thanks for your feedback! Development on the side-panel will resume in a few weeks once we've had the opportunity to collect feedback 👀 will keep this in mind

Compact view is certainly being considered and we've already got some designs for it

**Comment by ghost** [comment]
Nice! Very considerate of them to add new features for color blindness.

**Comment by weichen-lin-rooom** [comment]
Hope it can delete issue from repository, not only delete from project

**Reply by omerbensaadon** [reply]
thanks for your feedback! Development on the side-panel will resume in a few weeks once we've had the opportunity to collect feedback 👀 will keep this in mind

**Comment by ari-aviran** [comment]
Very nicely done.
Please add a `Copy issue link` button inside the panel

**Reply by iansan5653** [reply]
This is available now - thanks for the suggestion!

**Comment by lorenzwalthert** [comment]
While it works as expected, for me, I prefer to disable it (because I get information overflow having another feed in my starting page). Alternatively, get periodic updates in a batch instead of a feed as well as having the ability to filter the toipcs (i.e. some news are irrelevant for me).

**Comment by wouter-heerwegh** [comment]
I like this change, makes me use less tabs. It does not seem to populate the side panel if the author of the issue is a deleted user though.

**Reply by omerbensaadon** [reply]
thanks for your feedback! Development on the side-panel will resume in a few weeks once we've had the opportunity to collect feedback 👀 will keep this in mind

**Comment by jillesvangurp** [comment]
I'm not liking the disconnect between issues and projects. IMHO it is weird to have two completely separate issue trackers. Especially since the only way to sync them is via github actions. That's a hack, not a solution.

It should not impossible to have issues not connected to a project. How was that not a hard design goal from day 1? When projects was announced I thought "finally!" and then I was disappointed when it turned out projects were a completely separate thing. 

What I'm actually missing in Github regarding project management and task management. 

- A cross organizational view on issues. With more microservices than people having all these issue boards becomes tedious. The organizational projects thing did not address this because it exists separately from my issues. Again GH actions is not a solution here.

- Kanban boards for our issues (without the hacks via github actions. that needs to stop being a thing). I don't want to copy issues to a project. I just want to switch views on my issues.

- A more Asana like experience. Like Jira, github issues suffers from having a lot of click bureaucracy around common tasks. Asana nailed this. There's a lot not to like about Asana but this is one of the things they got right. The paged list with checkmarks feels very dated.

**Reply by nul800sebastiaan** [reply]
> A cross organizational view on issues. With more microservices than people having all these issue boards becomes tedious. The organizational projects thing did not address this because it exists separately from my issues. Again GH actions is not a solution here.

That's what project boards give you, you can add issues from multiple repositories on 1 board.

> Kanban boards for our issues (without the hacks via github actions. that needs to stop being a thing). I don't want to copy issues to a project. I just want to switch views on my issues.

I love the flexibility where I decide what goes on which board and through which criteria, defining that in actions is super powerful as I can run custom checks beforehand. And I believe the default configuration already lets you add each new created issue to a board. But maybe I missed your specific use case? 

![image](https://user-images.githubusercontent.com/304656/168603082-cdd57786-d555-4a82-892e-c90ea3a2437c.png)

**Reply by jillesvangurp** [reply]
> you can add issues from multiple repositories on 1 board

@nul800sebastiaan that's my point: I have to add them to a board. Why are the issues not there by default? I have hundreds of them. I don't want to micromanage issues and boards. I don't want to have custom github actions all over the place to automate this either.

**Reply by omerbensaadon** [reply]
thanks for your feedback! Development on the side-panel will resume in a few weeks once we've had the opportunity to collect feedback 👀 will keep this in mind

**Comment by airreader** [comment]
There is a problem when editing Japanese titles of issues and drafts from the side panel.

When entering Japanese text, the IME is used to convert the text into kanji characters.
However, when the conversion is confirmed, the entire title is saved.

This problem can be avoided by executing the following code in the Developer Tool as an example.
```
document.addEventListener('keydown',(e)=>{if(e.isComposing&&e.key==='Enter')e.stopImmediatePropagation()},1)
```

**Reply by omerbensaadon** [reply]
thanks for your feedback! Development on the side-panel will resume in a few weeks once we've had the opportunity to collect feedback 👀 will keep this in mind

**Comment by JakobJingleheimer** [comment]
This is awesome, especially after GitHub took away the old file list feature.

One small problem: once collapsed, it's very difficult to find the expand button again (it moves, albeit very close to where it was, but into an already busy UI area).

**Reply by omerbensaadon** [reply]
got any screenshots?

**Comment by d3adb5** [comment]
Pretty nifty feature! I believe this makes GitHub Projects more viable for business use, with teams opening issues being worked on in the side panel, without having to leave the board to focus on a specific task. A couple things I've observed:

- There's no mention of pull requests that mention a specific issue: unless there's a PR that "closes" the issue, it simply isn't mentioned in the side panel, whereas on the issue view itself the reference in the PR is displayed where conversation takes place.
- Similarly, there's no mention of changes being made to the issue, such as changing labels or assigning them.

It would be nice if PRs were also opened on the side panel. At least for me they are currently opening on a new tab.

**Reply by omerbensaadon** [reply]
thanks for your feedback! Development on the side-panel will resume in a few weeks once we've had the opportunity to collect feedback 👀 will keep this in mind

Sincerely appreciate your detailed feedback :)

**Reply by marzvrover** [reply]
:+1: here are a couple images that demonstrate how I find the context mentions create to be useful:

How they're viewed in issues:
<img width="1278" alt="Current issue mentions" src="https://user-images.githubusercontent.com/6617862/171864120-0703c0ef-b6ac-4c6e-977d-468c9073e89e.png">


Current Project Beta's sideview the context is lost:
<img width="1281" alt="Project Beta's issue view no mentions" src="https://user-images.githubusercontent.com/6617862/171864157-583faa54-d37a-4e0a-a584-202cf869920b.png">

**Comment by emin-rcrypto** [comment]
@omerbensaadon Is it possible in the Beta Project (at org level) to create a view where issues are listed per project.

You can do this in JIRA by selecting EPIC level view, so that each EPIC (Project's) issue is listed with swimlane columns. (pls see the attched img below)

<img width="1225" alt="image" src="https://user-images.githubusercontent.com/98781957/169748018-252fc23c-5010-4167-aed6-a0a4e38477eb.png">

**Reply by omerbensaadon** [reply]
This is out of scope for the side-panel but thanks for the feedback

**Reply by emin-rcrypto** [reply]
Correct, sorry wrong discussion group. Where best can I submit this request? :)

**Reply by omerbensaadon** [reply]
If you want to submit more general feedback, see: [github/feedback/discussions/categories/issues-feedback?discussions_q=sort%3Atop+created%3A%3E%3D2021-11-17+category%3A%22Issues+Feedback%22](https://github.com/github/feedback/discussions/categories/issues-feedback?discussions_q=sort%3Atop+created%3A%3E%3D2021-11-17+category%3A%22Issues+Feedback%22)

**Comment by hasak** [comment]
Can we leave comments on items, or ask additional questions regarding it (somewhere below description, for example)?

**Reply by iansan5653** [reply]
Hi, we don't support comments on draft issues, but you can convert the draft issue to a standard issue by using the **Convert to issue** button in the bottom right and selecting a repository to add it to. Then you can comment on it! 💬

**Comment by KAMALJAIS-ai** [comment]
Every each item need to have a detail function description. The side-panel is really a fantastic idea and it needs to be fine tuned with more advance integrated tools.

**Comment by efenaty** [comment]
The feature is really nice! I really love the table view. However, it would have been very nice if there were some sort of filters where we would be able to view all the issues assigned to a particular member or issues that are "In progress" or issues labelled with a particular label or a milestone.

**Reply by iansan5653** [reply]
✨ You can do that today by clicking the 'upside-down pyramid' icon to open the filter bar:
<img width="248" alt="Screenshot of the filter bar icon in projects beta." src="https://user-images.githubusercontent.com/2294248/170288718-f00c0068-ba0f-453f-adf1-c7c2a61a2ba1.png">

**Reply by omerbensaadon** [reply]
This feedback isn't relevant to the side-panel, if you want to submit more general feedback, see: [github/feedback/discussions/categories/issues-feedback?discussions_q=sort%3Atop+created%3A%3E%3D2021-11-17+category%3A%22Issues+Feedback%22](https://github.com/github/feedback/discussions/categories/issues-feedback?discussions_q=sort%3Atop+created%3A%3E%3D2021-11-17+category%3A%22Issues+Feedback%22)

**Comment by radrow** [comment]


**Reply by omerbensaadon** [reply]
This feedback isn't relevant to the side-panel, if you want to submit more general feedback, see: https://github.com/github/feedback/discussions/categories/issues-feedback?discussions_q=sort%3Atop+created%3A%3E%3D2021-11-17+category%3A%22Issues+Feedback%22

**Comment by johannes-manusvr** [comment]
We are missing certain features in this side-panel when using it on a project board.
- Transfer of issues from ones repository to another 
- Pull request linking

**Comment by anacarolinag** [comment]
O recurso é bom

**Comment by leighjboyd** [comment]
Side panel works great.  Before, it used to open new windows every time I clicked on an issue.  I would sometimes have 15 issues open at a time!  Now it stays neat and tidy.

**Comment by noklam** [comment]
I used the side panel and click "Convert to Issue" and it converts my draft into a blank issue 😅.

**Reply by omerbensaadon** [reply]
You didn't lose data did you?!

**Reply by omerbensaadon** [reply]
Do you mean that your blank draft issue became a blank actual issue?

**Reply by noklam** [reply]
Unfortunately, I did lose the data. I don't know how this happened, but I notice that I have the same page open in two tabs, so maybe because one of them is blank and it overrides it somehow? (I did click the convert button on the tab that has content)

**Reply by omerbensaadon** [reply]
strange... I'll see if we can investigate. I couldn't replicate this on my end 🤔

**Comment by ivanmayes** [comment]
Love the new sidebar view, our team immediately breathed a sigh of relief when released to not have to navigate through multiple tabs (especially those like me that have Projects pinned as browser tabs and then explode into a ton of extra tabs to navigate through).

We do have one issue in Projects around tracking Epics or an Epic-like issue that contains other tickets.  I know you can do this with the checkbox but often an epic ends up collecting other tickets as references instead of the top-down flow of starting at the epic and adding a checkbox so you can make it an issue.

It would be great if the sidebar view had a way to view issues that referenced the issue being viewed, as a way to track bottom-up flows of epic assignment.

Also open to any other Epic-like features that you may be dreaming up for the product!  Thanks!

**Reply by omerbensaadon** [reply]
@ivanmayes supporting Issue Hierarchies is in the works! Stay tuned :)

**Reply by shayfrendt** [reply]
Tasklists with hierarchy support have staff shipped!

https://github.com/github/engineering/discussions/1295

**Reply by MaxM-FF** [reply]
@shayfrendt That's great news! Thanks for the update and the discussions link! 

[Discussion Comment Link](https://github.com/orgs/community/discussions/16721#discussioncomment-2939031)
>[omerbensaadon](https://github.com/omerbensaadon) 
EDIT Sep 6: we have this in working software and our testing with Staff users now. I would anticipate getting access to this probably by the end of the year

**Comment by AmrSaber** [comment]
Love the side panel, but it would be more convenient if it marks the file as "completed" or "read" when I mark it as completed in the review.

**Comment by ClandestineClout** [comment]
I keep getting a little dot on my profile telling me to check the Feature Preview. I have done so dozens of times to try and get rid of this alert. It goes away temporarily when I do it but I'll come back a few days later and see it again, after which I have to go back and check the feature preview AGAIN in which NOTHING has changed. Please fix this very annoying issue.

**Comment by armandgrillet** [comment]
Hi, I have some feedback and I will mostly use that screenshot to give it:
<img width="1263" alt="PS 2022-06-08 à 12 26 54" src="https://user-images.githubusercontent.com/2117580/172594673-6039d523-5778-43f2-9d66-75288cba54e9.png">

1. The "Show all fields" is adding a second click whereas I have a lot of vertical space under this screenshot making more content viewable by default.
2. I don't see the issue reference in this view, this is annoying when sharing my screen with engineers and wanting them to dive in an issue ("The issue is 38607" being something I commonly say as a manager). Right now I have to copy the link and send it to my colleagues or open a new tab to get the reference.
3. When scrolling down the title of the issue does not stick at the top of the page, which is something great that happens when I am on an issue webpage and I scroll down:
<img width="928" alt="PS 2022-06-08 à 12 30 17" src="https://user-images.githubusercontent.com/2117580/172595180-d3bd0fc9-0cee-4219-95af-bf6a22b132f1.png">

Thanks for your work, those are only nits to make the UX better.

**Comment by Descent098** [comment]
I found a bug when creating a table inside a note it seems to be pulling alternate styling that makes it illegible:
![image](https://user-images.githubusercontent.com/22761705/172709415-d41e2026-fda5-462e-a2ff-97810943b8ba.png)

Here is the source markdown:

```md
List of types:

| filename | keywords | tasks | incidents | done |
|------------|------------|-------|-----------|-------|
| `access_requests.xlsx` | ['Add Account Access', 'Cannot access', 'Remove Account Access'] | True | False | ✅|
| `create_site_requests.xlsx` | ['Create New Website'] | True | False | ✅|
```

I do have the dark mode theme with tritanopia enabled on my github account and it seems like regular dark mode does not have this problem:

![image](https://user-images.githubusercontent.com/22761705/172709744-496a66db-af9f-4068-a01c-8073fe569585.png)



Likewise the status text is white on white which is also illegible:
![image](https://user-images.githubusercontent.com/22761705/172709447-9cf2e7e6-b08a-4a1a-9e97-cb2601a1bd12.png)

**Reply by iansan5653** [reply]
Hi, thanks for the bug report! I've forwarded this on to the team and we'll look into it.

**Reply by mattcosta7** [reply]
@Descent098 a fix for this was recently deployed.  Please let us know if you run into any further issues here, thanks!

**Reply by Descent098** [reply]
@mattcosta7 Looking good

![image](https://user-images.githubusercontent.com/22761705/181114459-09ad92ec-1091-44e4-aa44-6fee3a0e6782.png)

**Comment by qwc** [comment]
Hmm, this reminds me of JIRA boards and the opening of any issue on the right side.

That's a great thing, but guys, please be consistent and offer to open all issue types, PRs etc. in this side view, then the users do not have to switch tabs in their browser. At least I would like this a lot. :)

**Reply by omerbensaadon** [reply]
PRs is a top request! We hear you :)

**Comment by Descent098** [comment]
One other thing that would be nice to have is to be able to specify a few markdown-styles to the title of an issue/note. Specifically being able to bold, italicize and set `inline code blocks` would be handy since sometimes titles will include names of functions like the example below and delineating it would be handy: 


![image](https://user-images.githubusercontent.com/22761705/172938598-e6e24eb9-ad3f-4194-a510-129a3af9b77d.png)

**Reply by omerbensaadon** [reply]
I've opened an issue for this! Thank you

**Comment by LukeWolfGames** [comment]
It would also be good to change the background colour of the status and any other added fields to make certain items in fields more noticeable on the projects list.

**Comment by LukeWolfGames** [comment]
When viewing a project by milestone, any issues that are in the No Milestone category are not getting sorted say if you were to sort by ascending / descending on a custom added field. They are only sorting for the issues with a milestone assigned.

**Comment by eddiezane** [comment]
This PR crashes Chrome for me with the new side-panel enabled and opened. When it doesn't, clicking on filenames does.

https://github.com/kubernetes/kubernetes/pull/109111

**Reply by iansan5653** [reply]
Hi @eddiezane, thanks for the report. This is not related to Projects Beta, but I'll forward this to the Pull Requests team.

**Reply by iansan5653** [reply]
This should be fixed now! Please [file a bug report](https://support.github.com/contact/bug-report) if you still see this issue.

**Comment by ivanmayes** [comment]
This issue may be a bit tangential to this thread, but there is an issue when trying to assign outside collaborators in the Projects side panel.  Whether it is in a draft or issue state, I can't find certain outside collaborators, even though they are added to this repo and to the project.

![image](https://user-images.githubusercontent.com/847700/174094913-3de23d95-70ca-4f4a-9901-f92d1a2361f0.png)


Once I go to the issue page I can successfully add the outside collaborators.
![image](https://user-images.githubusercontent.com/847700/174094306-11a8d8a5-8a1e-4e52-92e5-2c2b441b99af.png)

This is within a Github Organization with SSO, not sure exactly what types of permission settings are flagged but that may have something to do with it.

**Reply by filip-stefanov** [reply]
Same issue here. So I used to refresh the browser and assign the issue to outside collaborators. After creating the issue, you should refresh the browser to assign the issue to the outside collaborators.
But that should be a bug for this feature. I am really bothering with assigning the issue to the team members. Hope this bug would be fixed asap. :) @omerbensaadon @ivanmayes

**Reply by omerbensaadon** [reply]
I'll open an issue to investigate this! Thanks for your feedback :)

**Comment by danemacmillan** [comment]
When an open side panel shows an issue from a repository, all of the commits that reference that issue should appear. Currently, opening a side panel in a project means I always need to click on the link to open the issue in a new tab to see the actual commit progress against the issue. I can only assume this is eventually being added, but if this was an oversight, it's worth mentioning how important those commit references are within an issue. Ultimately, I would expect everything from a fully-opened repository issue to be available in the project side panel.

**Reply by omerbensaadon** [reply]
We plan to implement this eventually but I don't have a timeline for ya. 

We hear you!

**Reply by nicola-whyte** [reply]
We've just moved to using projects as our main sprint board and we're pretty keen on this too please.

**Reply by ADTC** [reply]
@nicola-whyte IMHO GitHub Projects isn't designed with sprints in mind. We actually moved to Jira because we want to do sprints and GH Projects just wasn't working well to run our sprints effectively.

Jira has a lot of sprint-specific features and other powerful features like workflows and automation, none of them exist in GH Projects, which is just a very simple multi-repo issue tracker. The workflow/automation in GH is very simple and highly opinionated, not extremely flexible like the one in Jira. GH doesn't have any kind of story point tracking and we had to use labels for it which can only be filtered, while in Jira, story points is a first-class feature and it's used for velocity computation and report generation.

It's a shame, really, as I wanted to keep using GitHub Projects, but our needs greatly overgrew its capabilities.

Bonus tip: Jira has a free version, and it has dark mode in beta now.

**Comment by MiracleXYZ** [comment]
Embedding GeoJSON maps in issue **comments** causes infinite loading.

Steps to reproduce:

1. Open an issue and link it to the project.
2. Create a comment under the issue.
3. Embed the toy example of GeoJSON [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#using-geojson).
4. Open the issue on projects Side-Panel.

Note that only GeoJSON in **comments** causes this problem - embedding in the issue itself works fine.

**Reply by omerbensaadon** [reply]
I'll open an issue to investigate, thanks for your feedback!

**Reply by hussam-i-am** [reply]
Hi! Do you still see this issue? I just tested it out and it looks to be working for me now

<img width="1050" alt="Screen Shot 2022-07-26 at 5 12 01 PM" src="https://user-images.githubusercontent.com/584053/181112822-ab86e502-6e66-4098-b732-31e4fa39d23d.png">

**Reply by shiftkey** [reply]
:wave: @MiracleXYZ I've not been able to reproduce this either with the sample GeoJSON file - is there extra details that you can share to help us reproduce what you're seeing?

**Comment by Metrol** [comment]
I've tried moving our work flow on over to projects.  We've been using milestones to track at what stage of development an issue is in.  Not really thrilled with using milestones in this way, but it has distinct advantages over the project view.

For example, we have one milestone for `QA Review`.  Here, I can see all issues awaiting QA approval across any number of projects.  This is true of other milestones we use just for tracking.  Like seeing every issue in `Development`.  Having to drill down into individual projects doesn't provide what we need from a management perspective.

Moving an issue between milestones shows who did what and when.  From what I've seen, there is zero tracking on who did what on the project view.
-  Who added an issue to the project?  When?
-  Who moved an issue to a different status?  When? 
-  Can I notify people when an issue has moved to a specific status?

Every non-critical issue we have goes through the same basic workflow.  When I create a project, I would want to be able to define for my organization a template that would always be used.  A minor point compared to my first two.

Perhaps it's a beta thing, but I don't understand by every project is scoped at the organization level only.  I would want to be able to open a project that only gets worked on from within a specific repository.  When working within a repo, I really don't want to be presented with every unrelated repo in the org.  At the org level, it would also be helpful to know which repo the project is connected to.

My intention here is to provide what I hope is helpful criticism.  I really like the visual presentation for the project view.  I would love to move our workflow to utilize it.  Thing is, as awkward as using milestones for workflow may be, we'll be going back to that for now.  At least until something changes with my primary concerns.

- Ability to list all the issues for a given project status across multiple projects.
- Record who made status changes, and when were they made.
- Notification lists for each status in a project.
- Template a new project workflow.
- Project scoping to Org or Repo level.

**Reply by omerbensaadon** [reply]
Hey this is great feedback, I enjoyed reading it, but it is in the wrong place 

I would see if any of your requests exist in https://github.com/orgs/github-community/discussions/categories/issues and give those a +1, if not, opening a new discussion in the "Issues" category would be helpful for others to find it. 

Thanks for taking the time to write all that out!

**Reply by VladimirAlexiev** [reply]
@omerbensaadon and @Metrol I posted https://github.com/orgs/community/discussions/44406 which is a slightly related question.

**Comment by cory-baker** [comment]
I'd like the ability to add a label to an item without converting to an issue. As a project manager, I have no clue if the ticket is linked to the API or Web repo. And therefore would like to assign a label without having to create an issue in a repo first

**Comment by martinsifra** [comment]
**Text fields with URL's are not clickable links.** 

When you ad text field where you want to store URL, it's underlined as URL but when you click it, you start editing the field.

![url-field](https://user-images.githubusercontent.com/1210906/180224101-f798865c-bbb5-46ce-b5a4-f96f3bf10594.gif)

**Reply by omerbensaadon** [reply]
Hey actually you can do this with HTML, bit of a hidden feature: <a href={LINK}>click me</a>

Very sorry its taken me this long to respond, but it technically isn't relevant to this feature

**Comment by janpieterz** [comment]
It'd be awesome if the url could be rewritten when opening an issue into a side panel to deep-link to that issue, rather than that the url stays on the view. Fighting the muscle memory to just copy the url from the browser is hard.

**Reply by omerbensaadon** [reply]
We've heard this feedback before! It's on a long list of tech-debt to resolve but we want to move in this direction

**Comment by mschile** [comment]
When I add a label from the issue side-panel, the labeled event doesn't show up in the issue timeline both in the UI and from the timeline API. Am I missing something?

Example, `Needs Reproduction` label on https://github.com/cypress-io/cypress/issues/23607

**Reply by omerbensaadon** [reply]
Hmm... it should?

**Reply by mschile** [reply]
@omerbensaadon, it looks like https://github.com/community/community/discussions/30817 was also logged for the issue.

**Reply by omerbensaadon** [reply]
ooo good callout... I'll be sure to submit this bug

**Comment by omerbensaadon** [comment, marked as answer]
`This thread is no longer monitored by PM`


Hey folks, this has actually shipped globally for quite awhile, and the feedback on this Discussion is starting to just be general feedback for Projects. Please put feedback for Projects under "Issues" here: https://github.com/community/community/discussions/new?category=issues

**Comment by ADTC** [comment]
How do I move from issue to issue in the side panel? I find that I have to close the side panel and open it again for the next issue. It's inconvenient. There should be some arrow buttons (for mouse) or keyboard shortcuts like left/right arrows to just load the previous/next issue in the list inside the open side panel.

**Reply by Julian-Dumitrascu** [reply]
Do you refer to the window opened over project views to display an issue?
